bot/exts/info/pypi.py

```diff
@@ -31,16 +30,13 @@ def __init__(self, bot: Bot):
     @command(name="pypi", aliases=("package", "pack"))
     async def get_package_info(self, ctx: Context, package: str) -> None:
         """Provide information about a specific package from PyPI."""
-        embed = Embed(
-            title=random.choice(NEGATIVE_REPLIES),
-            colour=Colours.soft_red
-        )
+        embed = Embed(title=random.choice(NEGATIVE_REPLIES), colour=Colours.soft_red)
         embed.set_thumbnail(url=PYPI_ICON)
 
         error = True
 
-        if (character := re.search(ILLEGAL_CHARACTERS, package)) is not None:
-            embed.description = f"Illegal character passed into command: '{escape_markdown(character.group(0))}'"
+        if characters := re.search(ILLEGAL_CHARACTERS, package):
```

Comment:
Do you think it would be better to use `ILLEGAL_CHARACTERS.search`?
```suggestion
        if characters := ILLEGAL_CHARACTERS.search(package):
```

Reply:
I don't see how this would improve the code.

Reply:
> I don't see how this would improve the code.

Hmm I guess so.